components/dashboard/src/data/workspaces/update-workspace-description-mutation.ts

```diff
@@ -27,19 +27,14 @@ export const useUpdateWorkspaceDescriptionMutation = () => {
             // pro-actively update workspace description rather than reload all workspaces
             queryClient.setQueryData<ListWorkspacesQueryResult>(queryKey, (oldWorkspacesData) => {
                 return oldWorkspacesData?.map((info) => {
-                    if (info.workspace.id !== workspaceId) {
+                    if (info.id !== workspaceId) {
                         return info;
                     }
 
                     // TODO: Once the update description response includes an updated record,
                     // we can return that instead of having to know what to merge manually (same for other mutations)
```

Comment:
please update version in setup.ts that we invalidate caches.

We should do each time when we change shapes.